shiny/ui/_html_deps_external.py

```diff
@@ -15,14 +20,43 @@
 * shinyverse dependencies (e.g. bslib, htmltools), see `shiny.ui._html_deps_shinyverse`
 """
 
+ThemeProvider = Union[Tagifiable, HTMLDependency, list[HTMLDependency]]
+
+
+def bootstrap_theme_deps(theme: str | Path | ThemeProvider | None) -> TagList:
+    deps_bootstrap = bootstrap_deps(include_css=theme is None)
+
+    if theme is None:
+        deps_theme = None
+    elif isinstance(theme, str) and theme.startswith("http"):
```

Comment:
Up to you, but we could also support a double-slash prefix? https://mina86.com/2022/urls-support-slash-slash/